ui/src/app/components/app-router.tsx

```diff
@@ -42,7 +42,7 @@ export const AppRoute = ({path, data = {}, component: Component}): React.ReactEl
   const routeParams = useParams();
   const routeHistory = useHistory();
 
-  return <Route path={path} >
+  return <Route exact={true} path={path} >
```

Comment:
do we want this to work like this for ALL react routes?

Reply:
@petesantos what say you? I'm just happy it works at all.

Reply:
Let me test this - I intended to open another PR for this - my apologies for that slipping through.
In my original testing with react router, it seemed that exact routes are a bit more predictable.

Reply:
Tested - I don't see any issues. If we need to we can pass in a parameter to disable exact matching